### PR TITLE
Test release bugfixes

### PR DESCRIPTION
**Pull request by vankesteren**

### CODE REVIEW

File: JASP-Engine/JASP/R/exploratoryfactoranalysis.R [hunk 1]
```diff
@@ -82,18 +82,24 @@ ExploratoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
...
-      for (i in 1:(nVar-1)) {
-        for (j in (i+1):nVar) {
-          thisCor <- cor(dataset[,i],dataset[,j])
-          if(!is.na(thisCor) && thisCor == 1) allCorr <- allCorr + 1
-        }
+      P <- ncol(dataset)
+      
+      # check whether a variable has too many missing values to compute a 
+      # correlation
+      if (any(is.na(S))) {
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/exploratoryfactoranalysis.R]: Unless I'm missing it `S` is used before it's defined

Comment (vankesteren) [reply JASP-Engine/JASP/R/exploratoryfactoranalysis.R]: wow that's stupid of me. I'll fix ASAP

**Comment by TimKDJ** [comment]
Can you fix the unit test for missing values? Guess your NA check works now ;)

**Comment by TimKDJ** [comment]
With your latest commit you moved `S` again..

**Comment by vankesteren** [comment]
I commit too fast and you comment too fast ;)

**Comment by vankesteren** [comment]
should be good now!

**Comment by TimKDJ** [comment]
> 
> 
> I commit too fast and you comment too fast ;)

**Pull request merged**